components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -768,6 +788,10 @@ mojom::AutoGenerateQuestionsPref AIChatTabHelper::GetAutoGeneratePref() {
   return pref;
 }
 
+bool AIChatTabHelper::ShouldFetchPageContent() {
+  return !should_page_content_be_disconnected_;
+}
+
```

Comment:
IMO this doesn't need to be a function because 1) only reading one variable, 2) only used once, and 3) we already have a function named `MaybeGeneratePageText` whose purpose is already to be the single place to determine whether page content should be fetched.

Reply:
makes sense, removed